tensorflow/compiler/jit/xla_compilation_cache.cc

```diff
@@ -181,33 +181,37 @@ Status XlaCompilationCache::BuildExecutable(
 
 Status XlaCompilationCache::Compile(
     const XlaCompiler::Options& options, const NameAttrList& function,
-    absl::Span<const XlaCompiler::Argument> args,
+    std::vector<XlaCompiler::Argument>& args,
```

Comment:
I may have missed this, but why is this a non-const ref?

Btw, you could still keep this as a `Span` and copy it explicitly into an `std::vector`.